srcpkgs/remake/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'remake'
+pkgname=remake
+version=4.3
```

Comment:
this is not the correct version, as upstream versions based on the make version (4.3) and the remake version (1.6). something like `4.3+1.6` probably makes the most sense.

to transform that version to the upstream tag, `${version//+/+dbg-}`

Reply:
updated.